pkg/kube/client.go

```diff
@@ -476,10 +476,28 @@ func delete(c *Client, resources ResourceList, propagation metav1.DeletionPropag
 	mtx := sync.Mutex{}
 	err := perform(resources, func(info *resource.Info) error {
 		c.Log("Starting delete for %q %s", info.Name, info.Mapping.GroupVersionKind.Kind)
-		err := deleteResource(info, propagation)
-		if err == nil || apierrors.IsNotFound(err) {
-			if err != nil {
-				c.Log("Ignoring delete failure for %q %s: %v", info.Name, info.Mapping.GroupVersionKind, err)
+		if err2 := info.Get(); err2 != nil {
```

Comment:
`apierrors.NotFound` should be ignored here
In case of installing chart containing Jobs, the installation process will be failed
```suggestion
		if err2 := info.Get(); err2 != nil {
		if apierrors.IsNotFound(err2) {
				c.Log("Ignoring get failure for %q %s: %v", info.Name, info.Mapping.GroupVersionKind, err2)
				mtx.Lock()
				defer mtx.Unlock()
				res.Deleted = append(res.Deleted, info)
				return nil
			}
```